### DISCUSSION TITLE
ok..

### DISCUSSION THREAD

**Discussion by RealDealJinx** [DEMOLITIONDON96/Demolition-Engine#70] [General (Memes)]
![image](https://user-images.githubusercontent.com/88756260/164870280-b7928dfa-ce9f-4a68-b819-801467b8f1d5.png)
I NEVER HAD MAC :0 *well used to*

**Comment by ThatOneSillyGuyDon** [comment]
I see, you got Windows 11 as well

**Comment by Theoyeah** [comment]
when i have windows 10 :
![image](https://user-images.githubusercontent.com/97792861/164891619-6944eb8e-6866-400f-b336-f2771821b72d.png)
